build-php/resources/templates/README.mustache

```diff
@@ -0,0 +1,149 @@
+# Criteo Marketing SDK for PHP
+
+[![Build Status](https://travis-ci.com/criteo/criteo-php-marketing-sdk.svg?branch=master)](https://travis-ci.com/criteo/criteo-php-marketing-sdk)
+![Packagist Version](https://img.shields.io/packagist/v/criteo/criteo-php-marketing-sdk)
+
+This PHP package is automatically generated by the [OpenAPI Generator](https://openapi-generator.tech) project:
+
+- API version: {{appVersion}}
+{{#artifactVersion}}
+- Package version: {{artifactVersion}}
+{{/artifactVersion}}
+{{^hideGenerationTimestamp}}
+- Build date: {{generatedDate}}
+{{/hideGenerationTimestamp}}
+- Build package: {{generatorClass}}
+{{#infoUrl}}
+For more information, please visit [{{{infoUrl}}}]({{{infoUrl}}})
+{{/infoUrl}}
+
+## Requirements
+
+PHP 5.5 and later
```

Comment:
Here greater or equal 5.5

Reply:
Nice catch. That's from the official template. I'll update the README to match the composer file, i.e 7.1.